packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -506,8 +807,21 @@ export const colorThemeReadOnlyDisabledWithoutButtons: StoryFn = createFixedThem
 
 export const colorThemeReadOnlyDisabledWithButtons: StoryFn = createFixedThemeStory(
     createMatrix(component, [
-        [readOnlyStates[1]],
-        [disabledStates[1]],
+        [disabledReadOnlyStates[6]],
+        [actionButtonStates[1]],
+        leftIconStates,
+        errorStates,
+        typeStates,
+        appearanceStates,
+        fullBleedStates,
+        valueStates
+    ]),
+    colorThemeDarkGreenBackground
+);
+
+export const colorThemeReadOnlyDisabledAppearanceReadOnlyWithButtons: StoryFn = createFixedThemeStory(
+    createMatrix(component, [
+        [disabledReadOnlyStates[7]],
```

Comment:
If we think we'll need to break out this dimension again maybe we should go ahead and make some const aliases for states?

```ts
const disabledReadOnlyState = {
    none: disabledReadOnlyStates[0],
    appearanceReadOnly: disabledReadOnlyStates[1],
    //...
    readOnlyDisabledAppearanceReadOnly: disabledReadOnlyStates[7],
} as const;
```

Reply:
I like that suggestion. I'm not 100% sure if this was your suggestion or not, but I created those aliases in `states.ts` so we can easily use them from our other component matrix tests. I think that'll also make it really easy to use them from components that don't support the full matrix (e.g. select & combobox that don't have `readonly`).